src/ctapipe/image/tests/test_extractor.py

```diff
@@ -281,6 +282,17 @@ def test_extract_around_peak_charge_expected():
     assert_equal(charge, n_samples)
 
 
+def test_variance_extractor(toymodel):
+    _, subarray, _, _, _, _ = toymodel
+    # make dummy data with known variance
+    rng = np.random.default_rng(0)
+    var_data = rng.normal(2.0, 5.0, size=(5000, 2, 1855))
```

Comment:
The shape here is a bit weird for testing an image extractor. Of course for the test this doesn't really matter, but what you are doing here means

n_channels = 5000
n_pixels = 2 
n_samples = 1855